assets/pages/new-contributions-landing/contributionsLandingReducer.js

```diff
@@ -66,6 +69,9 @@ type FormState = {
   thankYouPageStage: ThankYouPageStage,
   hasSeenDirectDebitThankYouPageCopy: boolean,
   payPalHasLoaded: boolean,
+  isSignInRequired: boolean,
+  isIdentityRequestPending: boolean,
+  lastIdentityResponse: IdentityResponse,
```

Comment:
maybe this would be clearer what this is if it were named `lastIdentityResponseStatus`